examples/examples_py_test.go

```diff
@@ -75,6 +75,27 @@ func TestDefaultTagsPython(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestVpcPython(t *testing.T) {
+	// This test is not stable for several reasons so we can't run it in CI.
+	// Deletion of the aws-native:ec2:IpamPoolCidr regularly fails with "GeneralServiceException: Error occurred during operation 'The CIDR has one or more allocations.'" even after the VPC is deleted.
+	// Sometimes Update of the aws-native:ec2:IpamPool fails with "NotStabilized: IpamPoolCidrFailureReason(Message=The CIDR has one or more allocations."
+	// The refresh of the aws-native:ec2:IpamPool fails due to a diff in the provisionedCidrs property.
+	// It's also not ideal that we have to create the IPAM manually to avoid the test failing if run in parallel.
```

Comment:
We could just use another resource that includes properties in both lists to test, but honestly, I'd rather see a unit test than the an e2e test for this particular behavior.

Reply:
Yeah, I agree I think unit test is probably easier here. Though I think it'll take some further refactoring to make it accessible for testing.

I was finding it challenging to identify other affected resources but just managed to put together the folowing regexes to find 91 resources with the same pattern: 
`createOnlyProperties.*/properties/([^"]+)".*\n*.*writeOnlyProperties.*"/properties/\1"`
`writeOnlyProperties.*/properties/([^"]+)".*\n*.*createOnlyProperties.*"/properties/\1"`

```
aws-acmpca-certificate
aws-acmpca-certificateauthority
aws-apigateway-apikey
aws-appconfig-extensionassociation
aws-appconfig-hostedconfigurationversion
aws-applicationinsights-application
aws-appsync-sourceapiassociation
aws-backupgateway-hypervisor
aws-cloudformation-moduledefaultversion
aws-cloudformation-publictypeversion
aws-cloudformation-publisher
aws-cloudformation-typeactivation
aws-cognito-userpooluser
aws-connect-instance
aws-connect-phonenumber
aws-datasync-agent
aws-datasync-locationazureblob
aws-datasync-locationefs
aws-datasync-locationnfs
aws-datasync-locations3
aws-datasync-locationsmb
aws-datazone-datasource
aws-datazone-environment
aws-datazone-environmentprofile
aws-datazone-subscriptiontarget
aws-directoryservice-simplead
aws-ec2-eip
aws-ec2-instanceconnectendpoint
aws-ec2-keypair
aws-ec2-networkinsightsaccessscope
aws-ecr-pullthroughcacherule
aws-elasticache-serverlesscache
aws-elasticloadbalancingv2-truststorerevocation
aws-finspace-environment
aws-gamelift-build
aws-glue-schema
aws-grafana-workspace
aws-imagebuilder-component
aws-imagebuilder-containerrecipe
aws-imagebuilder-workflow
aws-iot-cacertificate
aws-iot-certificate
aws-iot-jobtemplate
aws-ivs-playbackkeypair
aws-memorydb-cluster
aws-neptunegraph-privategraphendpoint
aws-networkmanager-connectpeer
aws-oam-link
aws-omics-referencestore
aws-omics-sequencestore
aws-omics-workflow
aws-opensearchserverless-collection
aws-opensearchserverless-securityconfig
aws-pcaconnectorad-connector
aws-pcaconnectorad-directoryregistration
aws-pcaconnectorad-template
aws-redshift-cluster
aws-redshiftserverless-namespace
aws-refactorspaces-application
aws-refactorspaces-environment
aws-refactorspaces-route
aws-refactorspaces-service
aws-route53recoverycontrol-cluster
aws-route53recoverycontrol-controlpanel
aws-route53recoverycontrol-safetyrule
aws-s3-accessgrant
aws-s3-accessgrantsinstance
aws-s3-accessgrantslocation
aws-sagemaker-app
aws-sagemaker-appimageconfig
aws-sagemaker-dataqualityjobdefinition
aws-sagemaker-domain
aws-sagemaker-modelbiasjobdefinition
aws-sagemaker-modelexplainabilityjobdefinition
aws-sagemaker-modelpackage
aws-sagemaker-modelqualityjobdefinition
aws-sagemaker-userprofile
aws-ssm-document
aws-stepfunctions-statemachineversion
aws-systemsmanagersap-application
aws-transfer-certificate
aws-vpclattice-accesslogsubscription
aws-vpclattice-listener
aws-vpclattice-rule
aws-vpclattice-servicenetworkserviceassociation
aws-vpclattice-servicenetworkvpcassociation
aws-workspacesweb-browsersettings
aws-workspacesweb-identityprovider
aws-workspacesweb-ipaccesssettings
aws-workspacesweb-portal
aws-workspacesweb-usersettings
```